onnxruntime/python/tools/kernel_explorer/kernels/fast_gelu_test.py

```diff
@@ -0,0 +1,100 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import sys
+
+sys.path.append("../build")
```

Comment:
I think we should make ke searchable without this `sys.path` manipulation, but maybe in a separate PR.

Reply:
Yes, that would be nice. We can address in a separate PR.